src/CSSTransition.js

```diff
@@ -220,17 +215,6 @@ class CSSTransition extends React.Component {
     doneClassName && removeClass(node, doneClassName);
   }
 
-  reflowAndAddClass(node, className) {
-    // This is for to force a repaint,
-    // which is necessary in order to transition styles when adding a class name.
-    if (className) {
-      /* eslint-disable no-unused-expressions */
-      node && node.scrollTop;
```

Comment:
Why is the reflow being removed?

Reply:
The point is to avoid synchronous layout, which this forces. In my testing it was no longer necessary when using `fastdom` as the class manipulation happens on another tick.

Reply:
I'm a bit concerned about this, bc my previous testing trying to use rAF directly instead of the sync reflow led to animations not running. I wish I had a specific test case to try tho, I don't honestly remember what the cases were. Maybe during interrupts or fast toggling?

Reply:
I haven't been able to replicate any problems in my testing - ideas?

Reply:
theoretically, what is ensuring that `active` classes are being added on a different tick than the initial classes? From what I can tell here, every call to `fastdom.mutate` adds to the same queue, if the initial and active classes are added within the the rAF time frame (very likely?) they will flush on the same tick.

Reply:
To follow up on this, we've been using this in prod for 3mo+ without any issue. The forced reflow is unnecessary as fastdom will delay the change until the next animation frame.

Reply:
The `-active` classes work as expected, I tested this by adding `opacity: 0` to `-enter` and `opacity: 1` to `-enter-active`. The element faded in, which wouldn't happen if they were added in the same tick.

Reply:
The `-active` classes work as expected, I tested this by adding `opacity: 0` to `-enter` and `opacity: 1` to `-enter-active`. The element faded in, which wouldn't happen if they were added in the same tick.